module 4: intro-ComputeCanada/README.md

```diff
@@ -1 +1,66 @@
-Coming Soon
\ No newline at end of file
+# ComputeCanada from scratch
+
+ComputeCanada is a Canadian high-performance computing (HPC) infrastructure, aiming to provide computing and support for academics to advance research. The computing facilities aka Clusters are spread across Canada, and they are run by the Digital Research Alliance of Canada, through regional partners.
+
+The following guides walk you through step by step to be able to access those Cluster machines.
+
+### Sponsorship. 
+Access is enabled by sponsorship by a Canadian academic, with the only criterion being the sponsoree should have some sort of affiliation (e.g.: Student or Collaborator). To get a sponsorship,
+1. **Register for an Account**: Visit [Compute Canada Database](https://ccdb.alliancecan.ca/security/login) and register. When prompted, use Karim's CCRI: `kif-392-aa`.
+2. **Approval**: Once you have submitted your registration, notify Karim to check his email for your approval.
+
+### Access
+Getting a sponsorship grants access to all the Compute Canada clusters (e.g.: Beluga, Cedar). Go to the terminal, access the machine in the cluster through `ssh YOUR_NAME@CLUSTER_NAME.computecanada.ca`. 
```

Comment:
In  the next weeks, all computecanada.ca will be changed to the new account. Also I am not sure that new users get to use the "computecanada"